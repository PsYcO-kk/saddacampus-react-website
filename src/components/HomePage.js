import React, { Fragment } from 'react';
import VideoSection from './VideoSection.js'
import Services from "./Services.js";
import Blog from "./Blogs.js";
import UserReview from './UserReview.js';

const HomePage = () => (
	<Fragment>
		<VideoSection />
		<Services />
		<Blog />
		<UserReview />
	</Fragment>
);

export default HomePage;
