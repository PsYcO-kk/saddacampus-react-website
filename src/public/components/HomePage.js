import React, { Fragment } from 'react';

import LoginSection from './homepage/LoginSection.js';
import ServicesSection from "./homepage/ServicesSection.js";
import BlogsSection from './homepage/BlogsSection.js';
import ReviewsSection from './homepage/ReviewsSection.js';
import NewsSection from './homepage/NewsSection.js';
import GetAppSection from './homepage/GetAppSection.js';


const HomePage = () => (
	<Fragment>
		<LoginSection />
		<ServicesSection />
		<BlogsSection />
		<ReviewsSection />
		<NewsSection />
		<GetAppSection />
	</Fragment>
);

export default HomePage;
