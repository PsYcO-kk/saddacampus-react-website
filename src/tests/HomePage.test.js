import {shallow} from 'enzyme';
import React from 'react';
import HomePage from '../components/HomePage'

test("should render HomePage correctly" , ()=>{
    const wrapper = shallow(<HomePage />)
    expect(wrapper).toMatchSnapshot();
})
