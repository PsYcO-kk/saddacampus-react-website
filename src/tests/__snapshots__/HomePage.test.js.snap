// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing for HomePage component 1`] = `
<React.Fragment>
  <LoginSection />
  <VideoSection />
  <ServicesSection />
  <BlogsSection />
  <ReviewsSection />
  <GetAppSection />
</React.Fragment>
`;
